environment/sql/data.sql

```diff
@@ -1,108 +1,98 @@
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (9, 'PLAY', 'Plays/Queues a song from a link / a few keywords to search on YouTube', 1, 1, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (10, 'QUEUE', 'Display the songs currently in the queue', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (11, 'SKIP', 'Skip the current song', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (12, 'LOOPQUEUE', 'Toggle loop queue mode', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (13, 'NOWPLAYING', 'Display the currently playing song', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (14, 'CLEAR', 'Clear the music queue', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (15, 'RESUME', 'Resume the music player', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (16, 'PAUSE', 'Pause the music player', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (17, 'VOLUME', 'Sets volume for music player', 1, 0, 1);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (18, 'TOGGLECOMMAND', 'Toggles if a command is enabled or not', 1, 0, 1);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (19, 'SHUFFLE', 'Shuffles the queue', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (20, 'REMOVE', 'Removes a song from the queue', 1, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (21, 'MOVE', 'Move a song to a new queue index', 2, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (22, 'JUMPTO', 'Jump to an index of the queue', 1, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (23, 'ONJOINROLE', 'Sets a role to use on join', 1, 0, 1);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (24, 'FLIP', 'Flips a coin', 0, 0, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (25, 'TEAMS', 'Makes teams from the specified parameters', 3, 1, 0);
+INSERT INTO `command` (`id`, `name`, `description`, `paramCount`, `infiniteParam`, `level`) VALUES (26, 'UPDATEYTTOKEN', 'Sets the youtube tokens to a new value', 2, 0, 2);
```

Comment:
plz format so values are on diffrent line

Reply:
decided against it after discussion

Reply:
Not against it, but its up to the person writing the statements.